tests/oauth2/rfc6749/test_utils.py

```diff
@@ -108,3 +109,19 @@ def test_scope_to_list(self):
 
 
 
+class TestUrlValidation(TestCase):
+    def test_basic_urls(self):
```

Comment:
totally. there were zero tests so I wasn't sure what strategy you guys would prefer, so I went with the laziest one. I'll split these guys up.

what we should do is get some kind of better list of good/bad urls and have them iterated on.